README.md

```diff
@@ -6,12 +6,13 @@
 
 ![screenshot](./pakaMyPortfolio.png)
```

Comment:
- [ ] This screenshot is not working. Please have another look at it and solve the issue.